README.md

```diff
@@ -29,30 +35,31 @@ bundle install
 bundle exec rspec
 ```
 
-See [bundler.io](http://bundler.io) for the full documentation.
-
-### Troubleshooting
-
-For help with common problems, see [TROUBLESHOOTING](doc/TROUBLESHOOTING.md).
-
-Still stuck? Try [filing an issue](doc/contributing/ISSUES.md).
-
-### Other questions
+### Contributing
 
-To see what has changed in recent versions of Bundler, see the [CHANGELOG](CHANGELOG.md).
+While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together. So if you'd like to contribute to Bundler, that's awesome, and we <3 you!
 
-To get in touch with the Bundler core team and other Bundler users, please see [getting help](doc/contributing/GETTING_HELP.md).
+We have a [guide with recommended first steps](doc/contributing/README.md) that we suggest anyone interested in contributing to Bundler review first. Once you’ve completed those steps, feel free to get started contributing in any of the following ways:
 
-### Contributing
+- Adding new sections or making edits to the [documentation website](http://bundler.io) and [man pages](http://bundler.io/man/bundle.1.html)
+- Fixing typos
+- [Triage existing issues](doc/contributing/BUG_TRIAGE.md)
+- [Opening new issues](doc/contributing/ISSUES.md) (suggest feature requests, report new bugs)
+- [Reviewing pull requests](https://github.com/bundler/bundler/pulls)
+- [Backfilling unit tests](https://github.com/bundler/bundler/tree/master/spec/bundler) for modules that [lack coverage](https://codeclimate.com/github/bundler/bundler/coverage)
 
-If you'd like to contribute to Bundler, that's awesome, and we <3 you. There's a guide to contributing to Bundler (both code and general help) over in [our documentation section](doc/README.md).
+### Get support
 
-While some Bundler contributors are compensated by Ruby Together, the project maintainers make decisions independent of Ruby Together. As a project, we welcome contributions regardless of the author’s affiliation with Ruby Together.
+- **Troubleshooting**. If you have either `rubygems-bundler` or `open_gem` installed, please try removing them before filing an issue. For help with common problems, check out the [troubleshooting guide](doc/TROUBLESHOOTING.md).
+- **Reporting a bug**. If you’ve tried the troubleshooting guide and something is still not working, you can file an issue. Run `bundle-report-bug` to get all of the information you need to report a bug.
```

Comment:
@indirect can you confirm `bundle-report-bug` is the correct command? I remember seeing that in the doc we were both working off of